components/camel-jira/src/main/java/org/apache/camel/component/jira/producer/AddIssueProducer.java

```diff
@@ -25,6 +27,7 @@
 import com.atlassian.jira.rest.client.api.domain.IssueType;
 import com.atlassian.jira.rest.client.api.domain.Priority;
 import com.atlassian.jira.rest.client.api.domain.input.IssueInputBuilder;
+import com.google.common.base.Strings;
```

Comment:
Do not use google/guava for this - Camel has API for checking if a string is empty, or since Camel is now Java 11 based, you can use blank() method on Strings from the JDK itself

Reply:
org.apache.camel.util.ObjectHelper isEmpty method

As we need to backport to 3.14.x then we cannot use Java 11 APIs as 3.14 supports JDK8

Reply:
Fixed that, thanks.